codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/AcceptorModel.java

```diff
@@ -64,7 +61,14 @@ public void setMatcher(String matcher) {
     }
 
     public String getArgument() {
-        return argument;
+        String[] unformatted = argument.split("\\.");
+        StringJoiner formatted = new StringJoiner(".");
+
+        for (String format : unformatted) {
+            formatted.add(StringUtils.uncapitalize(format));
```

Comment:
Does this handle the case we talked about where DBInstance should be capitalized to dbinstance?